features/support/vcr.ts

```diff
@@ -70,9 +71,18 @@ Before(function (
     ) {
       fs.mkdirSync(path.dirname(frozen), { recursive: true });
       fs.writeFileSync(frozen, JSON.stringify(date));
+    } else {
+      throw new Error(`Time file '${frozen}' not found: create one setting \`RECORD=true\` or ignore it using \`RECORD=none\``);
```

Comment:
I think we may get there when we don't want to? For example if we're in mode ignore?I think we can ignore that one and just focus on the error you raised below.

Reply:
How about this?